ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -95,12 +95,48 @@ class Internals(SimpleNamespace):
 
 navigator_subcommds = [
     SubCommand(name="collections", description="Explore available collections"),
-    SubCommand(name="config", description="Explore the current ansible configuration"),
-    SubCommand(name="doc", description="Review documentation for a module or plugin"),
+    SubCommand(
+        name="config",
+        description="Explore the current ansible configuration",
+        epilog=(
+            "Note: With '--mode stdout', 'ansible-navigator config' additionally supports"
+            " the same parameters as the 'ansible-config' command."
+            " For more information about these, try "
+            " 'ansible-navigator config --help-config --mode stdout'"
+        ),
+    ),
+    SubCommand(
+        name="doc",
+        description="Review documentation for a module or plugin",
+        epilog=(
+            "Note: With '--mode stdout', 'ansible-navigator doc' additionally supports"
+            " the same parameters as the 'ansible-doc' command."
+            " For more information about these, try "
+            " 'ansible-navigator doc --help-doc --mode stdout'"
+        ),
+    ),
     SubCommand(name="ee-details", description="Explore execution environment details"),
-    SubCommand(name="inventory", description="Explore an inventory"),
+    SubCommand(
+        name="inventory",
+        description="Explore an inventory",
+        epilog=(
+            "Note: With '--mode stdout', 'ansible-navigator inventory' additionally supports"
+            " the same parameters as the 'ansible-inventory' command."
+            " For more information about these, try "
+            " 'ansible-navigator inventory --help-inventory --mode stdout'"
```

Comment:
`ansible-inventory` and `ansible-playbook` display the help even if no inventory/playbook is provided. Using the command suggested just displays:
```
[ERROR]: Command provided: 'inventory --help-inventory --mode stdout --ee false'
[ERROR]: An inventory is required when using the inventory subcommand
[ERROR]: Configuration failed, using default log file location: /home/liveuser/ansible-navigator.log. Log level set to debug
```

Reply:
ah!  I should have mentioned that, sorry....  Those 2 won't do anything until https://github.com/ansible/ansible-navigator/issues/159 is done... sorry

Reply:
I will raise a PR ready to add support for `--help-inventory` and `help-playbook` and will raise it soon